### ISSUE TITLE
[Bug]: Bottled app doesn't start anymore

### ISSUE THREAD

**Issue by petru-d** [bottlesdevs/Bottles#3534]
### Describe the bug

As title says, I was successfully running a windows app (Battle.Net) until yesterday.
Now it doesn't start anymore.

Bottles also gets stuck when I try to change the runner.

### To Reproduce

Start bottles.
Start bottle.

### Package

Flatpak from Flathub

### Distribution

Ubuntu 24.04.1 LTS

### Debugging Information

```shell
p@p-MS-7C02:~$ flatpak run --command=bottles-cli com.usebottles.bottles info health-check
Official Package: true
Version: '51.15'
DE/WM: ubuntu
Display:
    X.org: true
    X.org (port): :0
    Wayland: true
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.8.0-47-generic
Disk:
    Total: 8354471936
    Free: 8354299904
RAM:
    MemTotal: 15.6GiB
    MemAvailable: 13.4GiB
Bottles_envs: null
```

### Troubleshooting Logs

```shell
p@p-MS-7C02:~$ flatpak run com.usebottles.bottles
21:54:51 (INFO) Launching with URI: None 
21:54:51 (INFO) [Activate] request received. 
21:54:51 (INFO) Bottles Started! 
21:54:51 (INFO) Performing Bottles checks… 
21:54:51 (INFO) Dxvks found:
	 - dxvk-2.4.1
 
21:54:51 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.13
 
21:54:51 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.7.1
 
21:54:51 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.1
 
21:54:51 (INFO) Runners found:
	 - soda-9.0-1
	 - wine-proton-9.0-3-amd64
	 - wine-9.19-staging-amd64
	 - sys-wine-9.0
 
21:54:51 (INFO) Bottles found:
	 - Battle.Net
 
21:54:51 (INFO) Catalog installers loaded 
21:54:51 (INFO) Catalog dependencies loaded 
21:54:51 (INFO) Catalog components loaded 
21:54:56 (INFO) Setting Key sync=fsync for bottle Battle.Net… 
21:54:56 (INFO) There is no running wineserver. 
21:55:06 (INFO) Launching an executable… 
21:55:06 (WARNING) Windows path detected. Avoiding validation. 
21:55:06 (INFO) Using Wine Starter -- run 
21:55:06 (INFO) Using EasyAntiCheat runtime 
21:55:06 (INFO) Using BattlEye runtime 
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (2).

here I tried stopping the bottle 
21:55:59 (INFO) Using Wine debug tool -- kill_process 
..but bottles just hang and had to be terminated

When trying to run the bottle again, the output is slightly different towards the end:

p@p-MS-7C02:~$ flatpak run com.usebottles.bottles
21:56:07 (INFO) Launching with URI: None 
21:56:07 (INFO) [Activate] request received. 
21:56:07 (INFO) Bottles Started! 
21:56:08 (INFO) Performing Bottles checks… 
21:56:08 (INFO) Dxvks found:
	 - dxvk-2.4.1
 
21:56:08 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.13
 
21:56:08 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.7.1
 
21:56:08 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.1
 
21:56:08 (INFO) Runners found:
	 - soda-9.0-1
	 - wine-proton-9.0-3-amd64
	 - wine-9.19-staging-amd64
	 - sys-wine-9.0
 
21:56:08 (INFO) Bottles found:
	 - Battle.Net
 
21:56:08 (INFO) Catalog dependencies loaded 
21:56:08 (INFO) Catalog components loaded 
21:56:08 (INFO) Catalog installers loaded 
21:56:08 (INFO) Setting Key sync=fsync for bottle Battle.Net… 
21:56:09 (INFO) There is no running wineserver. 
21:56:13 (INFO) Launching an executable… 
21:56:13 (WARNING) Windows path detected. Avoiding validation. 
21:56:13 (INFO) Using Wine Starter -- run 
21:56:13 (INFO) Using EasyAntiCheat runtime 
21:56:13 (INFO) Using BattlEye runtime 

... and it hangs here
```

**Comment by petru-d** [comment]
.. after waiting for a while, some more logging info pops up:
```
018c:err:sync:RtlpWaitForCriticalSection section 00006FFFFFF9F440 "/home/runner/build_wine/wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 018c, blocked by 0130, retrying (60 sec)
018c:err:sync:RtlpWaitForCriticalSection section 00006FFFFFF9F440 "/home/runner/build_wine/wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 018c, blocked by 0130, retrying (60 sec)
018c:err:sync:RtlpWaitForCriticalSection section 00006FFFFFF9F440 "/home/runner/build_wine/wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 018c, blocked by 0130, retrying (60 sec)
018c:err:sync:RtlpWaitForCriticalSection section 00006FFFFFF9F440 "/home/runner/build_wine/wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 018c, blocked by 0130, retrying (60 sec)
018c:err:sync:RtlpWaitForCriticalSection section 00006FFFFFF9F440 "/home/runner/build_wine/wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 018c, blocked by 0130, retrying (60 sec)
019c:err:environ:run_wineboot boot event wait timed out
```

**Comment by mirkobrombin** [comment]
I cannot reproduce the bug, have no idea why it happen. Something that comes to your mind? Anything changed in your system?

**Comment by petru-d** [comment]
I don't remember exactly, it was a while ago. I think my system might have done some automatic updates, but I can't say what exactly changed.
In the meantime I created another bottle, installed battle net and moved the games from the broken bottle to the new one, which worked. After a while I deleted the old bottle, so I also can't reproduce the issue anymore.

I guess there's no way forward here idk. Feel free to close the issue if there is no way to progress.
